### PR TITLE
Meal Plan link Form

### PR DESCRIPTION
**Pull request by antoniavonto**
continuing from: https://github.com/gordon-cs/gordon-360-ui/pull/1941
The cards are in different sizes, though I cannot fix this with \<br />
Also the edit toggle looks too big compared to the letters

image
image

### CODE REVIEW

File: src/views/Home/components/DiningBalance/index.jsx [hunk 1]
```diff
@@ -1,4 +1,14 @@
-import { Button, Card, CardContent, CardHeader, Grid, Link, Typography } from '@mui/material';
+import {
```
Comment (jsenning) [inline src/views/Home/components/DiningBalance/index.jsx:1]: Why is this import statement expanded across multiple lines?

Comment (antoniavonto) [reply src/views/Home/components/DiningBalance/index.jsx:1]: It is automatically with the Prettier extension we have been using

**Comment by antoniavonto** [comment]
image

File: src/views/Home/components/DiningBalance/index.jsx [hunk 2]
```diff
@@ -138,9 +148,20 @@ const DiningBalance = () => {
...
         >
-          <Grid item>
+          <Grid item sx={10}>
             <Typography variant="body2" className={styles.label2}>
               {diningInfo.ChoiceDescription}
+              <IconButton
+                variant="body2"
+                title="Change Meal Plan"
+                className={styles.label2}
+                //className={styles.gc360_my_profile_edit_icon}
```
Comment (amos-cha) [inline src/views/Home/components/DiningBalance/index.jsx:158]:
```suggestion
                className={styles.label2}
                //className={styles.gc360_my_profile_edit_icon}
```
I think we might need a better name than label2, also delete the other style tag if not needed

**Review by amos-cha — DISMISSED** [review]

File: src/views/Home/components/DiningBalance/index.jsx (hunk 2 above)
Comment (antoniavonto) [reply src/views/Home/components/DiningBalance/index.jsx:158]: This naming was given by the style team and this style file we are getting from is being used in different components in the Home folder. I can still change it if you want to

Comment (antoniavonto) [reply src/views/Home/components/DiningBalance/index.jsx]: removed the other style tag

**Review by jsenning — APPROVED** [review]
LGTM

**Pull request merged**